pkg/kubelet/cm/devicemanager/device/types.go

```diff
@@ -0,0 +1,24 @@
+package device
```

Comment:
I think it’s not good to move this.
maybe you can use an alias?

Reply:
do you have any good advice? I am also considering how to name it here 😢

Reply:
If you do not reuse a new package,cause an error "import cycle not allowed"